bundles/org.openhab.binding.shelly/README.md

```diff
@@ -0,0 +1,514 @@
+The Thing will be re-initialized and the Thing type adjusted and this was successful.
+
+## Binding Configuration
+
+The binding has some global configuration options.
+
+| Parameter      |Description                                                       |Mandatory|Default                                         |
+|----------------|------------------------------------------------------------------|---------|------------------------------------------------|
+| defaultUserId  |Default user id for HTTP authentication when not set in the Thing |    no   |admin                                           |
+| defaultPassword|Default password for HTTP authentication when not set in the Thing|    no   |admin                                           |
+
+## Thing Configuration
+
+|Parameter         |Description                                                   |Mandatory|Default                                           |
+|------------------|--------------------------------------------------------------|---------|--------------------------------------------------|
+|deviceIp          |IP address of the Shelly device                               |    yes  |none                                              |
+|userId            |The user id used for HTTP authentication                      |    no   |none                                              |
+|password          |Password for HTTP authentication*                             |    no   |none                                              |
+|lowBattery        |Threshold for battery level. Set alert when level is below.   |    no   |20 (=20%), only for battery powered devices       |
+|updateInterval    |Interval for the background status check in seconds.          |    no   |1h for battery powered devices, 60s for all others|
+|eventsButton      |true: register event "trigger when a button is pushed"        |    no   |false                                             |
+|eventsPush        |true: register event "trigger on short and long push"         |    no   |false                                             |
+|eventsSwitch      |true: register event "trigger of switching the relay output"  |    no   |true                                              |
+|eventsSensorReport|true: register event "posted updated sensor data"             |    no   |true for sensor devices                           |
+|eventsCoIoT       |true: Listen for CoIoT/COAP events                            |    no   |true for battery devices, false for others        |
+
+Independent from the updateInterval setting the binding will perform a settings refresh from the device once per minute.
+There is no event from the device that the settings have be changed by the Shelly App so this is kind of a fallback to get updates at least once per minute. 
+
+
+
+## Channels
+
+### General Notes
+
+- Various channels are only visible for linking when you click Show More in the channel definition
```

Comment:
```suggestion
```

Reply:
user already asked this, because they are not familar with this concept - listen to the users

Reply:
Then please update the general documentation - this is not at all binding specific, so if people are not aware of this feature, they will have the same issue with 100 other bindings as well. It does not make sense to explain it at 100+ places and we must not start doing so.

Reply:
ol, I‘m pretty sure it is explained, but users don‘t read the general doc so they ask those question in the community threads. Could I add a link? Could you point me to it?

Reply:
Adding 100+ links on various binding pages isn't any better.
And we might have other UIs in future that do not have a "Show More" button and work differently. So in order to decouple binding docs from UI, just don't add any info or link here.